packages/minimongo/cursor.js

```diff
@@ -135,33 +135,42 @@ export default class Cursor {
    *                        `callback`.
    */
   forEach(callback, thisArg) {
-    if (this.reactive) {
-      this._depend({
-        addedBefore: true,
-        removed: true,
-        changed: true,
-        movedBefore: true,
-      });
-    }
```

Comment:
Why is this._depend removed from here?

It looks like this could break important behavior, even if it’s not showing up in tests. The tests pass, but it seems this part isn’t actually covered.